internal/schema/1.1/moved.go

```diff
@@ -0,0 +1,34 @@
+package schema
+
+import (
+	"github.com/hashicorp/hcl-lang/lang"
+	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
+)
+
+var movedBlockSchema = &schema.BlockSchema{
+	Description: lang.Markdown("Refactoring declaration to specify what address to move where"),
+	Body: &schema.BodySchema{
+		HoverURL: "https://www.terraform.io/language/modules/develop/refactoring#moved-block-syntax",
+		Attributes: map[string]*schema.AttributeSchema{
+			"from": {
+				Expr: schema.ExprConstraints{
+					schema.TraversalExpr{OfScopeId: refscope.DataScope},
```

Comment:
I couldn't find a mention of `data` blocks in the `moved` documentation. Only `module` and `resource`. `data` is supported as well?

Reply:
Looks like data source references are just ignored, so I'll remove this constraint and I also filed https://github.com/hashicorp/terraform/issues/31405